.github/workflows/push.yaml

```diff
@@ -7,13 +7,13 @@ on:
 
 jobs:
   push:
-    uses: stakater/.github/.github/workflows/push_container.yaml@v0.0.42
+    uses: stakater/.github/.github/workflows/push_container.yaml@v0.0.62
     with:
       DOCKER_FILE_PATH: Dockerfile
       RELEASE_BRANCH: main
     secrets:
-      CONTAINER_REGISTRY_URL: ${{ secrets.STAKATER_NEXUS_PREPROD_REGISTRY }}/stakater
-      CONTAINER_REGISTRY_USERNAME: ${{ secrets.STAKATER_NEXUS_PREPROD_USERNAME }}
-      CONTAINER_REGISTRY_PASSWORD: ${{ secrets.STAKATER_NEXUS_PREPROD_PASSWORD }}
+      CONTAINER_REGISTRY_URL: ghcr.io/stakater
+      CONTAINER_REGISTRY_USERNAME: stakater-user
+      CONTAINER_REGISTRY_PASSWORD: ${{ secrets.STAKATER_GITHUB_TOKEN }}
       SLACK_WEBHOOK_URL: ${{ secrets.STAKATER_DELIVERY_SLACK_WEBHOOK }}
-      GH_TOKEN: ${{ secrets.STAKATER_GITHUB_TOKEN }}
```

Comment:
I think this needs to stay as is, the token is used to push tag to this repo and add comments and so on, nothing related to stakater-ab organization from what I can see?

Reply:
Yes, you're right. I will revert this change.